source/declarations.tex

```diff
@@ -2025,10 +2025,10 @@
 \end{example}
 
 \pnum
-For a \grammarterm{placeholder-type-specifier} with
-a \grammarterm{type-constraint},
-if the type deduced for the placeholder does not satisfy its
-immediately-declared constraint\iref{temp}, the program is ill-formed.
+A \grammarterm{placeholder-type-specifier} with a \grammarterm{type-constraint}
+introduces the immediately-declared constraint\iref{temp}
+of the \grammarterm{type-constraint} for the type deduced for the placeholder;
+this constraint shall be satisfied.
```

Comment:
The "introduced" we have for template parameters is referencing the "introduces [...] constraint-expressions" in [temp.constr.decl]. That's not relevant in this case, I think, so I'm not sure it means anything to say that the immediately-declared constraint is introduced, and instead we should just be requiring the immediately-declared constraints to be satisfied. Maybe:

> For a *placeholder-type-specifier* with a *type-constraint*, the immediately-declared constraint ([temp]) of the *type-constraint* for the type deduced for the placeholder shall be satisfied.

Reply:
Applied.